### PR TITLE
List champions

### PR DESCRIPTION
**Pull request by rcmuniz1994**
For now, I'm just listing the champions. I would like some review by more experient Vue devs for learning with my mistakes.

Thanks. :D

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,15 +1,29 @@
 <template>
-  <img class="logo" alt="League of Legends logo" src="./assets/logo.png">
-  <ShowcaseTitle msg="LoL Champions Showcase"/>
+  <div id="#app">
```
Comment (gustavoFreireS) [inline src/App.vue]: the hashtag is not needed, did you put it thinking in a specific purpose?

File: src/app/store.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  }
+};
+
+const actions = {
+  retrieveChampions: ({ commit }) => {
+    fetch('http://ddragon.leagueoflegends.com/cdn/10.22.1/data/pt_BR/champion.json')
+      .then(response => response.json())
+      .then(champions => {
+        commit('setChampions', Object.keys(champions.data));
+        console.log(Object.keys(champions.data));
```
Comment (gustavoFreireS) [inline src/app/store.js]: remove unused console

**Review by gustavoFreireS — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 1 above)
Comment (rcmuniz1994) [reply src/App.vue]: Actually, I just kept the id that comes on the default boilerplate. I'll fix it.

File: src/app/store.js (hunk 2 above)
Comment (rcmuniz1994) [reply src/app/store.js]: I'm sorry. I'll remove it.

File: src/App.vue (hunk 1 above)
Comment (rcmuniz1994) [reply src/App.vue]: Done!

File: src/app/store.js (hunk 2 above)
Comment (rcmuniz1994) [reply src/app/store.js]: Done!

**Review by gustavoFreireS — APPROVED** [review]

**Pull request merged**